### PR TITLE
added makefile for deployment on neomad server

### PR DESCRIPTION
**Pull request by vinyll**
`make deploy` updates https://styleguide.neomad.org for those who have a neomad server access set up.

### CODE REVIEW

**Review by fuentesloic — APPROVED** [review]

**Comment by fuentesloic** [comment]
@vinyll you have to pull `cirlceci` branch and merge it inside, right ?

**Review by fuentesloic — CHANGES_REQUESTED** [review]
@vinyll you have to pull `cirlceci` branch and merge it inside, right ?

**Comment by vinyll** [comment]
> @vinyll you have to pull cirlceci branch and merge it inside, right ?

We don't _have to_, but it will be compliant.
Right now it works as is.

**Comment by fuentesloic** [comment]
as is we can't merge this PR cause circleCI

**Review by fuentesloic — DISMISSED** [review]

**Comment by fuentesloic** [comment]
@vinyll could you review ?

File: index.html
Comment (vinyll) [inline index.html]: This should now be `…url=/content/index.html`

**Review by vinyll — COMMENTED** [review]
I can't set a status as I'm the author of this PR.
You'll need to request your own changes!
(is this a Microsoft product to be so weird?! :p )

**Comment by vinyll** [comment]
We couldn't merge but once the other PR was approved, this one would have been rebased and passed ;)
Someone else than me will be able to approve this PR.
But it the "index.html" must be resolved first!

File: index.html
Comment (fuentesloic) [reply index.html]: true I forget the modif!

**Review by fuentesloic — APPROVED** [review]

**Pull request merged**
